API.md

```diff
@@ -167,6 +167,88 @@ In this example, differencify will launch a browser instance and share same brow
 ```
 In this example, after calling `result` function it will return the previous step result as an object.
 
+## Detailed Result Information
```

Comment:
You probably need to update [Additional methods on top of Puppeteer's Page class table](https://github.com/NimaSoroush/differencify/blob/master/API.md#additional-methods-on-top-of-puppeteers-page-class). It still says `toMatchSnapshot -> no argument`

toMatchSnapshot | no argument
-- | --




toMatchSnapshot | no argument
-- | --




toMatchSnapshot | no argument
-- | --




toMatchSnapshot | no argument
-- | --